src/invidious.cr

```diff
@@ -33,7 +33,7 @@ require "./invidious/jobs/**"
 CONFIG   = Config.load
 HMAC_KEY = CONFIG.hmac_key || Random::Secure.hex(32)
 
-PG_URL = URI.new(
+PG_URL = CONFIG.database_url || URI.new(
```

Comment:
This doesn't actually work, and I've spent an hour on this trying various options such as moving the determination of the DB URI to a function with if/case statements and using bare if/case statements here.

The task at hand is simple, "if CONFIG.database_url is not nil then use that, otherwise if CONFIG.db is not nil then use that, otherwise raise" but I can't seem to get it to work. I'd appreciate any help here.